integration_tests/requirements.txt

```diff
@@ -4,6 +4,5 @@ dbt-redshift>=1.3.0,<2.0.0
 dbt-postgres>=1.3.0,<2.0.0
 dbt-spark>=1.3.0,<2.0.0
 dbt-spark[PyHive]>=1.3.0,<2.0.0
-dbt-databricks>=1.3.0,<2.0.0
-
+dbt-databricks>=1.6.0,<2.0.0
 oscrypto @ git+https://github.com/wbond/oscrypto.git@d5f3437
```

Comment:
This can be removed now. Not sure if any of the other PRs remove this, but wanted to call it out.